### ISSUE TITLE
Unclear link to compatibility score makes me think that all versions are 100% compatible

### ISSUE THREAD

**Issue by rdunlop** [dependabot/feedback#19]
I received an e-mail which read:
```
Bumps [syntax](https://github.com/dblock/syntax) from 1.2.1 to 1.2.2.

- [Changelog](https://github.com/dblock/syntax/blob/master/CHANGELOG)
- [Commits](https://github.com/dblock/syntax/compare/v1.2.1...v1.2.2)

[Dependabot compatibility score](https://dependabot.com/compatibility-score.html)
```

I would click on the Changelog, to see the changelog, and if that wasn't sufficiently convincing I'd click on the Commits....and as a last resort, if I wanted to see what the Dependabot users thought of this upgrade, I'd click on "Dependabot compatibility score"....and see "100% compatible"

Only now, after having done this a few times to I realize that this link always reads "100% compatible" because it's describing the FEATURE of the compatibility score, not the `syntax v 1.2.2` rating.

I recommend:
- making this "marketing" page more clearly that (marketing)
or
- Removing this link from the e-mail
or
- make this link actually go to a "syntax 1.2.1 vs 1.2.2" webpage, which reflects the same value as that of the PR badge.

I recognized my mistaken understanding when I was reading the PR, which clearly shows a badge of "compatibility: unknown".

**Comment by greysteil** [comment]
Interesting. I made it a link because I figured people wouldn't understand how it was calculated, but the flow above certainly sounds more confusing that helpful.

Long term, I'd like the link to go to a collection of test results for public repos, but that's a way off. In the meantime, it would be pretty straightforward for me to make the value on the compatibility score page linked to the dependency update you clicked across from - is that what you were thinking with the third option above?
